app/code/Magento/Inventory/Model/OptionSource/SourceItemStatus.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model\OptionSource;
+
+use Magento\Framework\Data\OptionSourceInterface;
+
+/**
+ * Class SourceItemStatus
+ *
+ * @api
+ */
+class SourceItemStatus implements OptionSourceInterface
+{
+    /**#@+
+     * Source items status values
+     */
+    const SOURCE_ITEM_STATUS_OUT_OF_STOCK = 0;
+    const SOURCE_ITEM_STATUS_IN_STOCK = 1;
```

Comment:
>> We agreed that UNIQUE key would be added to SourceItem table (Source_id, SKU)
and that would be added in a separate module.
--
We have separate task for this
https://github.com/magento-engcom/magento2/issues/42